### PR TITLE
Release 0.5.0

### PR DESCRIPTION
**Pull request by kaklise**
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Bumped the build number (if the version is unchanged)
* [x] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -1,16 +1,16 @@
...
 source:
-  url: https://github.com/USEPA/WNTR/archive/0.4.2.zip
-  sha256: c76fb0f1b1c40c1df3122aa575de08f75a62497cc522a081ecb6929ffd8069e0
+  url: https://github.com/USEPA/WNTR/archive/0.5.0.zip
+  sha256: 21bb1fde619094e550e9fec338feb3f78268596f8a9db7cace69aa4c216ca216
 
 build:
-  number: 0
+  number: 1
   script: "{{ PYTHON }} -m pip install . --no-deps -vv"
```
Comment (michaelbynum) [inline recipe/meta.yaml:14]: I think we need to now specify that the extensions need built. We should be able to do it with the environment variable.

File: recipe/meta.yaml [hunk 2]
```diff
@@ -1,16 +1,16 @@
...
 
 source:
-  url: https://github.com/USEPA/WNTR/archive/0.4.2.zip
-  sha256: c76fb0f1b1c40c1df3122aa575de08f75a62497cc522a081ecb6929ffd8069e0
+  url: https://github.com/USEPA/WNTR/archive/0.5.0.zip
+  sha256: 21bb1fde619094e550e9fec338feb3f78268596f8a9db7cace69aa4c216ca216
 
 build:
-  number: 0
+  number: 1
```
Comment (michaelbynum) [inline recipe/meta.yaml:13]: I think the build number can go to 0 if the version changes.

File: recipe/meta.yaml (hunk 1 above)
Comment (kaklise) [reply recipe/meta.yaml:14]: By setting BUILD_WNTR_EXTENSIONS='true'?  where should that be defined?

Comment (michaelbynum) [reply recipe/meta.yaml:14]: Yes. I'm not sure yet. I'm looking.

Comment (kaklise) [reply recipe/meta.yaml:14]: https://docs.conda.io/projects/conda-build/en/latest/resources/define-metadata.html#use-environment-variables
```
build:
  number: 0
  script_env:
    - BUILD_WNTR_EXTENSIONS='true'
  script: "{{ PYTHON }} -m pip install . --no-deps -vv"
```
Does that look right?

Comment (michaelbynum) [reply recipe/meta.yaml:16]: That does look right. I can check the actions to be sure.

Comment (kaklise) [reply recipe/meta.yaml:16]: Looks like the env variable was set, but the build still fails.

**Pull request merged**